test/libsolidity/util/TestFileParser.cpp

```diff
@@ -248,19 +248,26 @@ string TestFileParser::parseIdentifierOrTuple()
 {
 	string identOrTuple;
 
-	if (accept(Token::Identifier))
+	auto parseArray = [&]() -> string
```

Comment:
Why do you create and return another string? You capture ``[&]`` everything anyways, so you can just keep using ``identOrTuple`` in the lambda and return nothing...